Appraisals

```diff
@@ -1,3 +1,7 @@
+appraise 'rails_6_devise_4' do
+  # use gemspec constraints
+end
+
 appraise 'rails_5_devise_4' do
   # use gemspec constraints
```

Comment:
The gemspec constraints will result in Rails 6 being installed now. So this combination needs to be constrained if we want it to actually test Rails 5. Latest Devise is still in the 4 series.

```suggestion
  gem 'actionmailer', '~> 5.0'
  gem 'actionpack', '~> 5.0'
  gem 'activerecord', '~> 5.0'
```

Reply:
Note: the Gemfile generated by Appraisal (`gemfiles/rails_5_devise_4.gemfile`) will change as a result of this, those changes need to be committed together with this change.